specs/decimal/decimal.md

```diff
@@ -197,66 +339,307 @@ Windows App SDK provides a native language decimal data type for C++ as the
+
+inline bool operator< (const DECIMAL& left, const DECIMAL& right);
+
+inline bool operator<=(const DECIMAL& left, const DECIMAL& right);
+
+inline bool operator> (const DECIMAL& left, const DECIMAL& right);
+
+inline bool operator>=(const DECIMAL& left, const DECIMAL& right);
+
+namespace Microsoft::Windows::Foundation
+{
+class decimal
+{
+public:
+    decimal() = default;
+    ~decimal() = default;
+
+    decimal(const decimal& value);
+    decimal(decimal&& value);
+    constexpr decimal(const DECIMAL& value);
+    decimal(bool value)l
+    decimal(char value);
+    decimal(std::int16_t value);
+    decimal(std::int32_t value);
+    decimal(std::int64_t value);
+    decimal(std::uint8_t value);
+    decimal(std::uint16_t value);
+    decimal(std::uint32_t value);
+    decimal(std::uint64_t value);
+    decimal(float value);
+    decimal(double value);
+    decimal(long value);
+    decimal(unsigned long value);
+    decimal(PCWSTR value);
+    decimal(PCWSTR value, const LCID locale);
+    decimal(const std::wstring& value);
+    decimal(const std::wstring& value, const LCID locale);
+#if defined(__WINSTRING_H_)
+    decimal(const HSTRING& value);
+    decimal(const HSTRING& value, const LCID locale);
+#endif // defined(__WINSTRING_H_)
+#if defined(WINRT_BASE_H)
+    decimal(const winrt::hstring& value);
+    decimal(const winrt::hstring& value, const LCID locale);
+#endif // defined(WINRT_BASE_H)
+
+    decimal& operator=(const decimal& value);
+    decimal& operator=(decimal&& value);
+    decimal& operator=(const DECIMAL& value);
+    decimal& operator=(bool value);
+    decimal& operator=(char value);
+    decimal& operator=(std::int16_t value);
+    decimal& operator=(std::int32_t value);
+    decimal& operator=(std::int64_t value);
+    decimal& operator=(std::uint8_t value);
+    decimal& operator=(std::uint16_t value);
+    decimal& operator=(std::uint32_t value);
+    decimal& operator=(std::uint64_t value);
+    decimal& operator=(float value);
+    decimal& operator=(double value);
+    decimal& operator=(long value);
+    decimal& operator=(unsigned long value);
+    decimal& operator=(PCWSTR value);
+    decimal& operator=(const std::wstring& value);
+#if defined(__WINSTRING_H_)
+    decimal& operator=(const HSTRING& value);
+#endif // defined(__WINSTRING_H_)
+#if defined(WINRT_BASE_H)
+    decimal& operator=(const winrt::hstring& value);
+#endif // defined(WINRT_BASE_H)
+
+    const DECIMAL& to_decimal() const;
+    bool to_bool() const;
+    char to_char() const;
+    std::int16_t to_int16() const;
+    std::int32_t to_int32() const;
+    std::int64_t to_int64() const;
+    std::uint8_t to_uint8() const;
+    std::uint16_t to_uint16() const;
+    std::uint32_t to_uint32() const;
+    std::uint64_t to_uint64() const;
+    float to_float() const;
+    double to_double() const;
+    long to_long() const;
+    unsigned long to_ulong() const;
+    std::wstring to_string() const;
```

Comment:
Consider `to_wstring` to match the naming and be explicit. Also, it seems likely that somebody is going to ask for something that returns CHAR-based strings.

Reply:
RECOMMEND: +1 rename to `to_wstring()`